spec/forms/candidate_interface/course_selection/closed_course_selection_step_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::CourseSelection::ClosedCourseSelectionStep do
+  subject { described_class.new.class.route_name }
+
+  describe '.route_name' do
```

Comment:
These methods have been added to maintain compatibility with existing continuous application routes.
These will be removed in future.